_ont/ont-huawei-ma5671a-root.md

```diff
@@ -155,14 +155,6 @@ mtd -e image1 write mtd5.bin image1
 fw_setenv committed_image 1
```

Comment:
Steps 11 and 12 can also be removed, perhaps even 10 should be removed which is not strictly necessary and causes confusion

Reply:
Agreed, let's drop those and then move them into a generic section under the parent section